src/test/scala/is/hail/expr/ir/CompileSuite.scala

```diff
@@ -658,4 +659,15 @@ class CompileSuite {
     val off = f(region)
     assert(TString.loadString(region, off) == "hello")
   }
+
+  @Test def testSelectFields() {
+    val ir = SelectFields(MakeStruct(FastSeq("foo"->I32(6), "bar"->F64(0.0))), FastSeq("foo"))
```

Comment:
our style is to have spaces separating arrows from their left and right arguments